README.md

```diff
@@ -19,5 +18,22 @@ its components.
 - Util: performance analyzer, version checker, numeric and Collection utils, log and exception utils etc.
 - Rpc: rpc component for inner module communication, currently it's based on [Sofa-RPC](https://github.com/sofastack/sofa-rpc)
 
+## Learn More
+
+The [doc page](https://hugegraph.apache.org/docs/) contains more information about hugegraph modules.
+
+And here are links of other repositories:
+1. [hugegraph-server](https://github.com/apache/incubator-hugegraph) (graph's core component - OLTP server)
```

Comment:
prefer to keep 'https://github.com/apache/hugegraph' since it will be redirected to incubator-hugegraph

Reply:
fine, update it

However, we need pay attention to the word `incubator` in code or POM file, it may be unsatisfactory/invalid.